src/main/java/com/datatorrent/netlet/AbstractClient.java

```diff
@@ -139,6 +139,7 @@ public void suspendRead()
   public void resumeRead()
   {
     key.interestOps(key.interestOps() | SelectionKey.OP_READ);
+    key.selector().wakeup();
```

Comment:
This was causing counter downstream operator in Benchmark test to become stuck and it's container termination. After introducing the fix, Benchmark works with select() in OptimizedEventLoop.